app/presenters/sufia/work_show_presenter.rb

```diff
@@ -2,6 +2,7 @@ module Sufia
   class WorkShowPresenter < ::CurationConcerns::WorkShowPresenter
     # delegate fields from Sufia::Works::Metadata to solr_document
     delegate :based_near, :depositor, :identifier, :resource_type, :tag, to: :solr_document
+    attr_reader :upload_set_id
```

Comment:
Where does this attribute get assigned a value?  Should this get indexed into the Work's solr_doc and delegated?  We'll have to address the workflow where a Work is the recipient of multiple UploadSets, but that is for future us to sort out.